addons/calendar/static/src/js/base_calendar.js

```diff
@@ -187,52 +190,44 @@ var CalendarNotification = Notification.extend({
         },
     },
 
-    init: function(parent, title, text, eid) {
-        this._super(parent, title, text, true);
-        this.eid = eid;
+    init: function(parent, message) {
+        this._super(parent, message.title, message.message, true);
+        this.eid = message.event_id;
+        this.message = message;
     },
 });
 
-WebClient.include({
-    get_next_notif: function() {
+var calendarAlarmManager = WebWidget.extend({
+    init: function(parent, options) {
         var self = this;
-
-        this.rpc("/calendar/notify")
-        .done(function(result) {
-            _.each(result, function(res) {
-                setTimeout(function() {
-                    // If notification not already displayed, we create and display it (FIXME is this check usefull?)
-                    if(self.$(".eid_" + res.event_id).length === 0) {
-                        self.notification_manager.display(new CalendarNotification(self.notification_manager, res.title, res.message, res.event_id));
-                    }
-                }, res.timer * 1000);
-            });
-        })
-        .fail(function(err, ev) {
-            if(err.code === -32098) {
-                // Prevent the CrashManager to display an error
-                // in case of an xhr error not due to a server error
-                ev.preventDefault();
-            }
-        });
+        this._super(parent);
+        this.bus = bus.bus;
+        this.bus.last = 0;
```

Comment:
Is there any specific reason to set last = 0, note that, if last is 0 then first poll will return messages of last 50 seconds(by default)